source/jormungandr/jormungandr/street_network/handimap.py

```diff
@@ -54,10 +54,7 @@
     'url': 'https://www.handimap.fr',
 }
 
-
-class Languages(Enum):
-    french = "fr-FR"
-    english = "en-EN"
+LANGUAGE_TRANSFORMATION_LIST = {"en-US": "en-EN", "en-GB": "en-EN", "fr-FR": "fr-FR"}
```

Comment:
As stated in another comment, this "en-EN" language tag was a documentation error on our side (Handimap).
You can safely replace it with a more conventional `en-US` without any risk (`en-US` is our default, and already used used as fallback for any unknown language tag like `en-EN`).
Sorry for that documentation fail.

Reply:
> As stated in another comment, this "en-EN" language tag was a documentation error on our side (Handimap). You can safely replace it with a more conventional `en-US` without any risk (`en-US` is our default, and already used used as fallback for any unknown language tag like `en-EN`). Sorry for that documentation fail.

Hi @jthiard will it be possible to put the link of the DOC Hadimap as I don't have other than  https://dev.keolis-rennes.handimap.fr/docs/ and it give mes [ fr-FR, en-EN ] as language ?
Thanks

Reply:
Hi @kadhikari sure, we have two urls : 

-  https://dev.keolis-rennes.handimap.fr/docs/ : our staging instance
-  https://keolis-rennes.handimap.fr/docs/ our prod instance

At this time the two use the same version since there is no future version in staging yet.

If you still see `en-EN` on https://dev.keolis-rennes.handimap.fr/docs/ it may be your browser cache. You can try https://keolis-rennes.handimap.fr/docs/ which may be not cached on your side.

I just changed the webserver config to send "no cache" headers for our openAPI file to avoid this cache problem in the future.